test/coverage_scalatest_resources/consumer/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("//scala:scala.bzl", "scala_library", "scala_test")
+
+scala_library(
+    name = "consumer",
+    srcs = ["src/main/scala/com/example/consumer/Consumer.scala"],
+    deps = ["//test/coverage_scalatest_resources/resource"],
+)
+
+scala_test(
```

Comment:
Can test depend on lib instead of sources? eg.:

```
scala_test(
    name = "tests",
    srcs = [
        "src/test/scala/com/example/consumer/ConsumerSpec.scala",
    ],
    deps = [":consumer"],
)
```

Reply:
Indeed, it does work!  It alters the diff a bit, but in a good way.  Pushing this change now.